### PR TITLE
Minor fixes

### LINKED ISSUE #245: cleanup package distribution files
the current build/distribution contains almost the entire git repo and is very bloated

should clean this up using `setup.cfg` or `MANIFEST.in` configuration

### PR DESCRIPTION
**Pull request by CagtayFabry**
## Changes
minor fixes and cleanup
- clean up distribution files
- run pytest with source dist from pypi build

## Related Issues
Closes #245 

## Checks

- [ ] updated CHANGELOG.md
- [ ] updated tests
- [ ] updated doc/
- [ ] update example/tutorial notebooks

### CODE REVIEW

**Review by vhirtham — APPROVED** [review]

**Comment by marscher** [comment]
The testing requirements are not installed ;) Should we put them into a requirements-tests.txt file for pip inside the tests folder for easier installation?

**Comment by CagtayFabry** [comment]
> 
> 
> The testing requirements are not installed ;) Should we put them into a requirements-tests.txt file for pip inside the tests folder for easier installation?

I tried puuting them into `setup.cfg` but this messed up the conda build somehow 😕 https://github.com/BAMWelDX/weldx/pull/246/commits/2034908ae8afcc703e611a0366a46a7e30559174

I think it would be good to put requirements-tests.txt somewhere to keep track of the different plugins

**Comment by CagtayFabry** [comment]
ready for review as pytest now runs from the `sdist` in the build action @marscher

**Review by marscher — APPROVED** [review]

File: MANIFEST.in [hunk 1]
```diff
@@ -1 +1,10 @@
 recursive-include weldx/asdf/schemas *.yaml
+
+prune .binder
+prune .github
+prune conda.recipe
+prune doc
```
[the remark this answers is not in the document]
Comment (CagtayFabry) [reply MANIFEST.in:6]: there is `pip install -e .` aplenty in the actions

File: doc/conf.py [hunk 2]
```diff
@@ -41,7 +41,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "weldx"
-copyright = "2020, BAM"
```
[the remark this answers is not in the document]
Comment (CagtayFabry) [reply doc/conf.py:44]: I thought about that too, maybe another time :wink:

**Pull request merged**